kratos/python/add_factories_to_python.cpp

```diff
@@ -21,29 +21,34 @@
 #include "python/add_factories_to_python.h"
 #include "factories/linear_solver_factory.h"
 #include "factories/preconditioner_factory.h"
+#include "factories/register_factories.h"
 
 namespace Kratos
 {
 
 namespace Python
 {
 
+typedef UblasSpace<double, CompressedMatrix, boost::numeric::ublas::vector<double>> SpaceType;
+typedef UblasSpace<double, Matrix, Vector> LocalSpaceType;
+typedef LinearSolver<SpaceType,  LocalSpaceType> LinearSolverType;
+typedef TUblasSparseSpace<std::complex<double>> ComplexSpaceType;
+typedef TUblasDenseSpace<std::complex<double>> ComplexLocalSpaceType;
+
+typedef LinearSolverFactory< SpaceType, LocalSpaceType > LinearSolverFactoryType;
```

Comment:
wait why `LinearSolverFactoryType` is templated with the spaces? This should be templated with the `LinearSolverType` (which is templated with the spaces) I would say

Reply:
See the following comment